lib/tapioca/dsl/compilers/active_record_typed_store.rb

```diff
@@ -95,17 +95,18 @@ class ActiveRecordTypedStore < Compiler
         sig { override.void }
         def decorate
           stores = constant.typed_stores
-          return if stores.values.flat_map(&:accessors).empty?
+          return if stores.values.all? { |store| store.accessors.empty? }
 
           root.create_path(constant) do |model|
             stores.values.each do |store_data|
-              store_data.accessors.each do |accessor|
-                field = store_data.fields[accessor]
+              store_data.accessors.each do |accessor, name|
+                field = store_data.fields.fetch(accessor)
```

Comment:
Does this change means we're not going to be retro-compatible with `activerecord-typedstore < 1.5.0`?

Reply:
But this change here going from `fields[accessor]` to `fields.fetch(accessor)` is not strictly necessary.